pkg/apis/wmc/v1alpha1/windowsmachineconfig_types.go

```diff
@@ -0,0 +1,61 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// WindowsMachineConfigSpec defines the desired state of WindowsMachineConfig
+type WindowsMachineConfigSpec struct {
```

Comment:
Should [Windows build number](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/#handling-multiple-windows-versions-in-the-same-cluster) be added as a field here? 
According to the link:
> The Windows Server version used by each pod must match that of the node

It seems that there are multiple builds (versions) for Windows Server 2019. Based on that it would seem that we should keep note of the build number so that we would know if a pod could be scheduled on the node or not due to version reasons.

Reply:
As of now, the focus is on getting the Windows Server 2019 to the cluster(that's why I am not exposing the image to be used as of now). We may not go to the granular level of matching builds in future but we can always increase the surface area of api if we want to.